com.valvesoftware.Steam.yml

```diff
@@ -51,6 +51,7 @@ finish-args:
   - --env=MESA_GLSL_CACHE_DIR=
   - --env=STEAM_RUNTIME_PREFER_HOST_LIBRARIES=
   - --env=STEAM_RUNTIME=
+  - --env=STEAM_FLATPAK_XDG_DIRS_PREFIX=~/.var/app/com.valvesoftware.Steam
```

Comment:
Probably we shouldn't occupy `STEAM_` prefix?

Reply:
Might as well just XDG_DIRS_PREFIX. This takes precedence over user environment variables.

Reply:
But this is fine. I don't think it's harmful to just keep as is.

Reply:
I believe Steam can have special treatment of it's own variables. Maybe `FLATPAK_STEAM_XDG_DIRS_PREFIX`?

Reply:
Changed to `FLATPAK_STEAM_XDG_DIRS_PREFIX` just in case.